cfme/infrastructure/pxe.py

```diff
@@ -574,6 +574,39 @@ def delete(self, cancel=False, *ct_objs):
             else:
                 navigate_to(ct_obj, 'Details')
 
+    def copy(self, ct_obj, name=None, description=None, cancel=False):
+        """
+         This method is used to copy a Customization Template server via UI.
+
+         Args:
+             ct_obj : This field is the object of Customization Template.
+             name (str): This field contains the name of the newly copied Customization Template.
+             description (str) : This field contains the description of the newly
+              copied Customization Template.
+             cancel (bool): It's used for flag to cancel or not the copy operation.
+        """
+
+        view = navigate_to(ct_obj, 'Details')
+        view.toolbar.configuration.item_select("Copy this Customization Template")
+        view = ct_obj.create_view(PXECustomizationTemplateAddView)
+        if name is not None:
```

Comment:
**REQUIRED:** I think the following section should be modified as we skip if the fill is ```None```
```python
        if name is not None:
            view.fill({'name': name})
        if description is not None:
            view.fill({'description': description})
        else:
            description = 'Copy of {}'.format(ct_obj.description)
            view.fill({'description': description})
            name = 'Copy of {}'.format(ct_obj.name)
```
to
```python
        description = description or 'Copy of {}'.format(ct_obj.description)
        view.fill({'name': name, 'description': description})
        name = description
```